### PR TITLE
Mount gcloud kubeconifg to breeze

### PR DESCRIPTION
**Pull request by turbaszek**
---
**^ Add meaningful description above**

Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#pull-request-guidelines)** for more information.
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in [UPDATING.md](https://github.com/apache/airflow/blob/master/UPDATING.md).

### CODE REVIEW

File: backport_packages/setup_backport_packages.py [hunk 1]
```diff
@@ -215,7 +215,7 @@ def get_long_description(provider_package_id: str) -> str:
     :return: content of the description (README file)
     """
     package_folder = get_target_providers_package_folder(provider_package_id)
-    with open(os.path.join(package_folder, "README.md"), encoding='utf-8') as file:
+    with open(os.path.join(package_folder, "README.md"), encoding='utf-8', mode="w+") as file:
```
Comment (potiuk) [inline backport_packages/setup_backport_packages.py]: Unrelated? Separate PR?

**Review by potiuk — APPROVED** [review]
LGTM, but I think the README fix in backports should be separated out.

File: backport_packages/setup_backport_packages.py (hunk 1 above)
Comment (turbaszek) [reply backport_packages/setup_backport_packages.py]: Yeah, I will split this one

**Review by potiuk — APPROVED** [review]

File: scripts/ci/docker-compose/forward-credentials.yml [hunk 2]
```diff
@@ -29,3 +29,4 @@ services:
       - ${HOME}/.azure:/root/.azure:cached
       - ${HOME}/.config:/root/.config:cached
       - ${HOME}/.docker:/root/.docker:cached
+      - ${HOME}/.kube:/root/.kube:cached
```
Comment (turbaszek) [inline scripts/ci/docker-compose/forward-credentials.yml]:
```suggestion

```
I think we should remove this just for security reasons

**Review by potiuk — APPROVED** [review]

**Pull request merged**